website_snippet_country_dropdown/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Snippet's dropdown country VAT code
+===================================
+
+This module adds a template with a dropdown and an input text field, is a base
+for be inherited by others modules into a html form.
+
+Usage
+=====
+
+To extends this template you need inherit *country_dropdown* template and adds
+you personal code.
+The template have two input text, you can create your own variable for these
+inputs and pass it to the template with the name *vat_field* and
```

Comment:
You should use a more neutral name for the field associated. You can use it for VAT, or for another thing like telephone numbers. I'm thinking that for this being useful and generic enough for other cases, you should receive a parameter telling which res.country field do you want to use (code, name, a new field for phone prefix...), and another parameter for what is the destination field. This way:

* It's a real snippet.
* Can be inserted programatically or visually in any place.
* Can be used for a lot of things.